### PR TITLE
ISSUE 1250: add config

### LINKED ISSUE #1250: Add config for DFS storage and raft meta storage, e.g. a local dir.

### PR DESCRIPTION
**Pull request by SGZW**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

solve #1250 

## Changelog

- Improvement

## Related Issues

Fixes #1250 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by SGZW** [comment]
cc @drmingdrmer

**Comment by drmingdrmer** [comment]
> cc @drmingdrmer

What you need is just a click:DDD
image

**Comment by PsiACE** [comment]
> What you need is just a click:DDD

Contributors do not have sufficient permissions and may need to be handed over to datafuse bot in the future.

**Comment by drmingdrmer** [comment]
> > What you need is just a click:DDD
> 
> Contributors do not have sufficient permissions. It may need to be completed by datafuse-bot in the future.

Hm... Sorry I did not not yet notice that.

File: fusestore/store/src/configs/config.rs [hunk 1]
```diff
@@ -137,6 +137,14 @@ pub struct Config {
                       " Otherwise this argument is ignored.")
     )]
     pub id: NodeId,
+
+    #[structopt(
+        long,
+        env = "FUSE_STORE_LOCAL_FS_DIR",
+        help = "local dir for dfs storage and raft meta storage",
```
Comment (drmingdrmer) [inline fusestore/store/src/configs/config.rs:144]: Local fs dir is for storing files upload by users. 
All meta or raft data is in `meta_dir`

File: fusestore/store/src/tests/service.rs [hunk 2]
```diff
@@ -94,6 +94,9 @@ pub fn new_test_context() -> StoreTestContext {
     let t = tempdir().expect("create temp dir to store meta");
     config.meta_dir = t.path().to_str().unwrap().to_string();
 
+    let t = tempdir().expect("create local fs dir to store data");
+    config.local_fs_dir = t.path().to_str().unwrap().to_string();
+
```
Comment (drmingdrmer) [inline fusestore/store/src/tests/service.rs:99]: A tempdir guard needs to be held in `StoreTestContext`.
As the guard is dropped, the temp dir will be removed.

Similar to `meta_temp_dir`, here there should be another field holding the `local_fs_dir` guard.

File: fusestore/store/src/configs/config.rs (hunk 1 above)
Comment (SGZW) [reply fusestore/store/src/configs/config.rs:144]: ok，i will fix this

**Review by drmingdrmer — APPROVED** [review]
LGTM! Thanks man!

**Pull request merged**